python/cudf_polars/cudf_polars/dsl/expressions/datetime.py

```diff
@@ -138,6 +144,16 @@ def do_evaluate(
             for child in self.children
         ]
         (column,) = columns
+        if self.name == TemporalFunction.Name.ToString and plc.traits.is_timestamp(
+            column.obj.type()
+        ):
```

Comment:
What if the name is tostring but it's not a timestamp type? Or is that impossible? If it's impossible then checking just the name is sufficient